roles/common/tasks/serverspec.yml

```diff
@@ -1,8 +1,12 @@
 ---
+- name: clean all serverspec definition
```

Comment:
This will remove and recreate the file on every Ursula run. It might be okay if we are going to remove this as soon as we have Inspec checks for Ubuntu.